bemani/backend/jubeat/clan.py

```diff
@@ -1103,8 +1093,14 @@ def handle_gameend_final_request(self, request: Node) -> Node:
 
         return Node.void('gameend')
 
-    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score]) -> Node:
-
+    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score], mdata_ver: Optional[int]) -> Node:
+        if mdata_ver is None:
+            mdata_ver = 1
+        min_music_id, max_music_id = {
+            1: (0, 60000000),
+            2: (60000000, 90009999),
+            3: (90009999, 1000000000),
```

Comment:
This seems a bit arbitrary? Wouldn't that bias scores into different buckets depending on how new the game is?